Cython/Compiler/ModuleNode.py

```diff
@@ -599,15 +599,17 @@ def generate_type_definitions(self, env, modules, vtab_list, vtabslot_list, code
         for entry in vtabslot_list:
             self.generate_objstruct_predeclaration(entry.type, code)
         vtabslot_entries = set(vtabslot_list)
+        ctuple_cnames = set()
```

Comment:
It was unclear to me whether it would have been appropriate to add the `ctuple`s to `vtabslot_entries`, so I made another set.